recipe/meta.yaml

```diff
@@ -26,14 +26,14 @@ outputs:
         - setuptools >=77.0.3
         - setuptools-scm >=8
       run:
-        - python >={{ python_min }}
+        - python >={{ python_min }},<3.14
         - cartopy >=0.21
         - cf-units >=3.1
         - cftime >=1.5
-        - dask-core >=2025.1.0
+        - dask-core >=2025.1.0,<2025.10.0
         - libnetcdf !=4.9.1
         - matplotlib-base >=3.5,!=3.9.1
-        - netcdf4
+        - netcdf4 <1.7.3
```

Comment:
what's the reason for this pin, folks? Something we should be concerned of/with too?

Reply:
perfect, thanks, folks :beer: